### DISCUSSION TITLE
can use in windows xp with python 3.4?

### DISCUSSION THREAD

**Discussion by WingDust** [xonsh/xonsh#5169] [Q&A]
## xonfig

<details>

```
$ xonfig
```

</details>

### Traceback (if applicable)

<details>

```
traceback
```

</details>

## For community
⬇️  **Please click the 👍 reaction instead of leaving a `+1` or 👍  comment**

**Comment by jnoortheen** [comment]
Are you referring to installing xonsh with python 3.4?
